src/App.js

```diff
@@ -1,7 +1,47 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import Navigation from './Navigation'
+import CreatePage from './pages/CreatePage'
+import PoolPage from './pages/PoolPage'
 import userData from './userData'
-import CardList from './components/CardList'
+import { loadFromLocal, saveToLocal } from './utils'
 
 export default function App() {
-  return <CardList userData={userData} />
+  const [profiles, setProfiles] = useState(
+    loadFromLocal('profiles') || userData
+  )
+
+  return (
+    <Router>
+      <AppGrid>
+        <Navigation />
+        <Switch>
+          <Route path="/profile-pool">
```

Comment:
Maybe use `<Route exact path={["/","/profile-pool"]}` to link from url root to profile page as well?